src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  flex-direction: row;
+  display: flex;
+  gap: 4px;
+
+  &__star {
+    background-image: url('../images/star-active.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+
+    &:last-child {
+      background-image: url('../images/star.svg');
```

Comment:
Using `:last-child` to change the background image of the last star might not be the best approach if the number of active stars can vary. Consider using a more dynamic method, such as adding a specific class to inactive stars, to ensure the correct stars are styled as active or inactive.